<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>签名工具</string>
  </property>
  <widget class="QDateEdit" name="dateEdit_limit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="textRW_ID">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>600</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_sign">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>签名</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textRW_sig">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>601</width>
     <height>120</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_verify">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>检查</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>601</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>结果</string>
   </property>
   <widget class="QTextBrowser" name="textR_ID">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>600</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textR_limit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textR_sig">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>600</width>
      <height>60</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pbtn_localID">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>20</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>获取本地ID</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbtn_clear">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
